### ISSUE TITLE
Autofill for android

### ISSUE THREAD

**Issue by Puspharaj** [adar2378/pin_code_fields#129]
Nice work by the team!

I'm not sure if the package supports autofill, it seems not included in this current version. Is there any way to get it work?

Moreover android supports SMS verify, but that comes with bit restriction like, <#> Your otp code is 987879, If the sender doesn't contain # in front of the message, it won't get detected. 

I'm using Firebase OTP login that doesn't include # In front of every message, so no way of autofill that sms to my field.

Is there any solution to this? Thanks.

**Comment by adar2378** [comment]
`enablePinAutofill ` use this attribute, thanks

**Comment by Puspharaj** [comment]
Indeed, it was set default to true, but never works!!

**Comment by CherukuriSiva** [comment]
@Puspharaj  Can't we use https://pub.dev/packages/sms_autofill/example library here... to autofill the OTP?

**Comment by Puspharaj** [comment]
Nope, I tried it, won't work either for my case, evry otp has to start with #. In my case firbase won't work like that.

**Comment by adar2378** [comment]
@Puspharaj Then maybe you need to create your own custom solution for this, I'm afraid.

**Comment by omidraha** [comment]
Does this `enablePinAutofill`  attribute just work for IOS?

**Comment by harshmsomani** [comment]
Any update?
